### ISSUE TITLE
How can I plot my graphs with t higher than 1?

### ISSUE THREAD

**Issue by adsmendesdaniel** [analysiscenter/pydens#10]

**Comment by akoryagin** [comment]
Hey, @adsmendesdaniel!

My guess is you're asking about plotting-funcs. In fact, all of them accept an argument for a grid of points. For instance, plotter `plot_pair_1d` supports `points`-arg, which can include values larger than 1. In a similar manner, `plot_sections_3d` accepts `timestamps`-arg. 

Hope that helps!
Alex.

**Comment by adsmendesdaniel** [comment]
Looking at the first example:

`#load_ext autoreload
#autoreload 2

import os
import warnings
import sys

warnings.filterwarnings('ignore')
from tensorflow import logging
logging.set_verbosity(logging.ERROR)
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'

import numpy as np
import tensorflow as tf
from tqdm import tqdm_notebook
import matplotlib.pyplot as plt

sys.path.append('..')
from pydens import Solver, NumpySampler, cart_prod
from pydens import plot_loss, plot_2d, plot_pair_1d, plot_sections_2d, plot_sections_3d

from pydens import add_tokens
add_tokens()

# describing pde-problem in pde-dict
pde = {'n_dims': 1,
       'form': lambda u, t: D(u, t) - 2 * np.pi * cos(2 * np.pi * t),
       'initial_condition': 1 # will be transformed into callable returning 1
      }

# put it together in model-config
config = {'pde': pde,
          'track': {'dt': lambda u, t: D(u, t)}}

# uniform sampling scheme
s = NumpySampler('uniform')

# train the network on batches of 100 points
dg = Solver(config)
dg.fit(batch_size=150, sampler=s, n_iters=1500, bar='notebook')

plot_loss(dg.loss, color='cornflowerblue')

# plot real solution and network approximation
plot_pair_1d(dg,
             lambda t: np.sin(2*np.pi*t)+1,
             confidence=0.15,
             alpha=0.2)

# plot approximation of solution-derivative (porquê??)
plot_pair_1d(dg,
             lambda t: 2*np.pi*np.cos(2*np.pi*t),
             fetches='dt',
             confidence=0.6,
             alpha=0.2,
             ylabel=r'$\frac{d \hat{u}}{d t} | \frac{d u}{d t}$')
`
This plots a graph with 't' from 0 to 1. How can I use the arg "points" to plot the graph with 't' from 0 to 2?

**Comment by adsmendesdaniel** [comment]
A similar question: At the classical wave equation example code...:

import os
import warnings
import sys

warnings.filterwarnings('ignore')
from tensorflow import logging
logging.set_verbosity(logging.ERROR)
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'

import numpy as np
import tensorflow as tf
from tqdm import tqdm_notebook
import matplotlib.pyplot as plt

sys.path.append('..')
from pydens import Solver, NumpySampler, cart_prod
from pydens import plot_loss, plot_2d, plot_pair_1d, plot_sections_2d, plot_sections_3d

from pydens import add_tokens
add_tokens()

pde = {'n_dims': 2,
       'form': lambda u, x, t: D(D(u, t), t) - D(D(u, x), x),
       'initial_condition': [lambda x: x * (1 - x), 0],
       'time_multiplier': 'polynomial'}

body = {'layout': 'faR fa fa+ f',
        'units': [10, 25, 10, 1],
        'activation': [tf.nn.tanh, tf.nn.tanh, tf.nn.tanh]}

config = {'body': body,
          'pde': pde,
          'track': {'dt': lambda u, x, t: D(u, t)}}

s = NumpySampler('u') & NumpySampler('e', scale=1).truncate(3.0)

dg = Solver(config)
dg.fit(batch_size=200, sampler=s, n_iters=12, bar='notebook')

plot_loss(dg.loss, color='cornflowerblue')

timestamps = (0., 0.14, 0.28, 0.42, 0.56, 0.7)

plot_sections_2d(dg, timestamps=timestamps, grid_size=(2, 3), ylim=(-0.2, 0.3),
                 title='"Plucked string" (wave PDE in $\mathcal{R}$): approximate solution')

How can I display the graphs in a grid of 1x6 or 6x1? Just changing the line to "grid_size(1,6)" or "grid_size(6,1)" doesnt work.

Also, how can I plot just one single graph with this code? Setting "timestamps=(0.)" and "grid_size=(1,1)" doesnt work.

Sorry for asking too many questions, Im having a hard time with your plot functions. :)
